DPPP/test/unit/tPreFlagger.cc

```diff
@@ -21,32 +21,36 @@
 //
 // @author Ger van Diepen
 
-#include <lofar_config.h>
-#include <DPPP/PreFlagger.h>
-#include <DPPP/Counter.h>
-#include <DPPP/DPInput.h>
-#include <DPPP/DPBuffer.h>
-#include <DPPP/DPInfo.h>
-#include <Common/ParameterSet.h>
-#include <Common/StringUtil.h>
 #include <casacore/casa/Arrays/ArrayMath.h>
 #include <casacore/casa/Arrays/ArrayLogical.h>
 #include <casacore/casa/Arrays/ArrayIO.h>
 #include <iostream>
 
-using namespace LOFAR;
+#include <boost/test/unit_test.hpp>
+
+#include "../../PreFlagger.h"
+#include "../../Counter.h"
+#include "../../DPInput.h"
+#include "../../DPBuffer.h"
+#include "../../DPInfo.h"
+#include "../../../Common/ParameterSet.h"
+#include "../../../Common/StringUtil.h"
+
+using namespace DP3;
 using namespace DP3::DPPP;
 using namespace casacore;
 using namespace std;
 
+BOOST_AUTO_TEST_SUITE(preflagger)
```

Comment:
If we do this we also need to rename all the methods and classes that are defined in the tests because they will violate unique naming constraints e.g. multiple execute() methods or TestInput class names.